chapters/data-analysis.tex

```diff
@@ -231,28 +232,34 @@ \section{De-identifying research data}
 
 \section{Cleaning data for analysis}
 
-Data cleaning is the second stage in the transformation of data you received into data that you can analyze.\sidenote{\url{
-https://dimewiki.worldbank.org/Data\_Cleaning}}
-The cleaning process involves (1) making the data set easily usable and understandable,
+Data cleaning is the second stage in the transformation of raw data into data that you can analyze.
+The cleaning process involves (1) making the data set easy to use and understand,
 and (2) documenting individual data points and patterns that may bias the analysis.
 The underlying data structure does not change.
 The cleaned data set should contain only the variables collected in the field.
 No modifications to data points are made at this stage, except for corrections of mistaken entries.
 
 Cleaning is probably the most time-consuming of the stages discussed in this chapter.
-This is the time when you obtain an extensive understanding of the contents and structure of the data that was collected.
-Explore your data set using tabulations, summaries, and descriptive plots.
-You should use this time to understand the types of responses collected, both within each survey question and across respondents.
+You need to acquire an extensive understanding of the contents and structure of the raw data.
+Explore the data set using tabulations, summaries, and descriptive plots.
 Knowing your data set well will make it possible to do analysis.
 
-\subsection{Correcting data entry errors}
+\subsection{Identifying the identifier}
 
-There are two main cases when the raw data will be modified during data cleaning.
-The first one is when there are duplicated entries in the data.
+The first step in the cleaning process is to understand the level of observation in the data (what makes a row), 
+and what variable or set of variables uniquely identifies each observations.
 Ensuring that observations are uniquely and fully identified\sidenote{\url{https://dimewiki.worldbank.org/ID\_Variable\_Properties}}
 is possibly the most important step in data cleaning.
-Modern survey tools create unique observation identifiers.
-That, however, is not the same as having a unique ID variable for each individual in the sample.
+It may be the case that the variable expected to be the unique identifier in fact is either incomplete or contains duplicates. 
+This could be due to duplicate observations or errors in data entry.
+It could also be the case that there is no identifying variable, or the identifier is a long string, such as a name.
+In this case cleaning begins by carefully creating a numeric variable that uniquely identifies the data. 
```

Comment:
```suggestion
In this case cleaning begins by carefully creating a variable that uniquely identifies the data. 
```

Does it have to be **numeric**? I previously though so, and I still prefer that, but we had a discussion about this once, and no one had good arguments to why "A12345", "8765YZ" would not be permitted. They are less memory efficient, but not to a degree where it has any impact whatsoever to the vast majority of projects.